data/remnant/remnant missions.txt

```diff
@@ -4634,12 +4628,412 @@ mission "Remnant: Expanded Horizons Storms 1"
+			choice
+				`	"Do you think it could have something to do with the ion storms?"`
+					goto storms
+				`	"Could a similar 'change' have created the unstable wormholes?"`
+					goto wormholes
+
+			label storms
+			`	"It is certainly possible. There is a large flux of energy being poured into the surrounding nebulae, and the source, whatever it may be, has been active for eons. Who knows what kind of effect it may have originally had on these stars, millions of years ago?"`
+				goto end
+
+			label wormholes
+			`	"That is one possibility. Whichever civilization formed the destabilized wormholes -and we are nearly certain that they are artificial- would easily have the power to alter or even destroy entire stars. While this is one of the more unlikely scenarios, if it does happen to be the correct one, we would naturally be very concerned."`
+
+			label end
+			`	"In any case, the data we gathered will be invaluable towards our further understanding of not only the Ember Wastes, but the wider galaxy and its history. Depending on what we find in it, I might contact you in the future for further expeditions to exotic stars." You say farewell and leave, collecting your logistics allotment of <payment>.`
+
+
+
+mission "Remnant: Expanded Horizons Novas 2"
+	name "Supernova Remnant Comparison"
+	description "Investigate an anomalous supernovae remnant near Remnant space, then collect baseline astronomical data from the Saquergen system. Lastly, travel to the anomalous novae remnants near Korath space and return with the data to <destination>."
+	passengers 1
+	source
+		government "Remnant"
+	waypoint
+		system "Tenebra"
+	waypoint
+		system "Saquergen"
+	waypoint
+		system "Kasikfar"
+	waypoint
+		system "Peresedersi"
+	waypoint
+		system "Fasitopfar"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Novas 1: done"
+		random < 15
+	on offer
+		require "Research Laboratory"
+		conversation
+			`	The moment you finish your landing checks, making sure your ship is ready to launch at any notice, you receive a communication from Dawn. "Hello, Captain. We have finished sorting through the data collected in our previous expedition, and are ready to survey some additional systems that may shed light on the history of the Ember Wastes. Are you interested in transporting me?"`
+			choice
+				`	"Where are we headed to this time?"`
+				`	"Sorry, I'm busy right now."`
+					defer
+
+			`	"Well, our recent examination of the Wolf-Rayet stars has added further evidence supporting a theory we have had for nearly two centuries now. Rather than being some product of nature, there is clear evidence of tampering. While we knew these stars were the wrong mass and age to be in that sequence of stellar evolution, analysis of the flare events we observed has concluded that there is some kind of excess energy source powering the stars, keeping them in the Wolf-Rayet stage for far longer than should be possible."`
```

Comment:
```suggestion
			`	"Well, our recent examination of the Wolf-Rayet stars has added further evidence supporting a theory we have had for nearly two centuries now. Rather than being some product of nature, there is clear evidence of tampering. While we knew these stars were the wrong mass and age to be in that sequence of stellar evolution, analysis of the flare events we observed has concluded that there is some kind of excess energy source powering the stars, keeping them in the Wolf-Rayet stage for far longer than should be possible.`
```